Method add 추가, 테스트코드 추가

LTTTTTE opened the pull request:
- add 관련 테스트코드 추가
- contains indexOf 등 메소드들 테스트코드 추가

LTTTTTE committed:
Method add 추가, 테스트코드 추가

- add 관련 테스트코드 추가
- contains indexOf 등 메소드들 테스트코드 추가

lns13301 reviewed (commented):
코드를 압축시키는 방법을 배울 수 있는 기회가 되었습니다.
저의 코드 제작에도 참고하여 코드를 압축할 수 있도록 하겠습니다.
수고하셨습니다.

lns13301 reviewed (commented):
테스트코드에 Generic표기를 안하셨군요!

LTTTTTE committed:
Generic 표기 수정 , addIndex 테스트 코드 먼저추가

- addIndex 메소드 작성 전 테스트 코드 부터 작성완료
- 테스트 코드 내에 Generic 표기없이 클래스 생성 사용했던 내용 수정
- size 대신 array.length 를 사용하는게 뭔가 맞을거같아서 수정
